bool catchBlockPts
---
bool CatchObj
string blockName
geometry_msgs/PoseArray BlockPts